### DISCUSSION TITLE
Gesture to expand notification & other apps for buttons

### DISCUSSION THREAD

**Discussion by ghost** [jkuester/unlauncher#105] [Ideas]
Would love to have notifications expand with a swipe down gesture. Especially when hiding the status bar as it takes extra swiping to expand. This would also be more consistent with stock Android.  

Quick buttons
1. The setting button ought to be for system settings especially since there's a dozen ways already to bring up Unlauncher's settings. 😀
2. Would be nice to be able to select a different app, contacts or sms, etc.. some of us rarely use the phone or camera. 🤷

An idea for this area is, like [Olauncher](https://github.com/tanujnotes/Olauncher) , to use swipe left or right to trigger preferred apps instead of being locked into the icons that lock the user into one choice — ala slim launcher minimalistic mindset. But I guess that's up to what you are going for — locking down to minimums or lightweight yet fully functional.

BTW, the design and flow is very well done, very pleasing to the eye, etc. However I keep using Olauncher over Unlauncher because of
1. Gesture to expand notifications
2. Easy fast access to app's system settings
3. Not locked into certain app choices
4. Double tap to lock screen
5. Easier access to hidden apps

Mainly #1&4 🤷 

If you all are open to more suggestions I've got a few as I've been playing a lot with every text based launcher 😀, but to be fair, Unlauncher is pretty exceptional right now.

**Comment by maneater2** [comment]
I agree with olauncher's swiping feature, it would be pretty nice to have!
